tests/test_arcaflow_plugin_pcp.py

```diff
@@ -162,6 +162,21 @@ def test_functional_full(self):
                     fail=lambda _: self.fail("Output failed schema validation"),
                 )
 
+    def test_functional_fast_stop(self):
+        input = pcp_plugin.PcpInputParams(
+            pmlogger_interval=5.0,
+                    pmlogger_metrics="kernel.all.cpu.user mem.util.used",
+                    timeout=1,
+        )
```

Comment:
It would be nice to have a code comment explaining what this test is setting up and checking.  (I presume that we are asking PCP for a "long" interval and then killing it before that interval has completed, such that we "successfully" get no data...?)

Reply:
This was intended to test the case that triggers the retry loop ... but it doesn't actually do that. I couldn't massage the test condition to trigger the failure based on the way I documented the problem previously, unfortunately.

Reply:
So...you're going to remove this test, then?  😈 

If we're going to retain this test, could you add a comment describing what it _does_ do which is of value to us.